src/FpdiTrait.php

```diff
@@ -0,0 +1,263 @@
+<?php
+
+namespace Mpdf;
+
+use setasign\Fpdi\PdfParser\CrossReference\CrossReferenceException;
+use setasign\Fpdi\PdfParser\Filter\AsciiHex;
+use setasign\Fpdi\PdfParser\Type\PdfHexString;
+use setasign\Fpdi\PdfParser\Type\PdfIndirectObject;
+use setasign\Fpdi\PdfParser\Type\PdfNull;
+use setasign\Fpdi\PdfParser\Type\PdfNumeric;
+use setasign\Fpdi\PdfParser\Type\PdfStream;
+use setasign\Fpdi\PdfParser\Type\PdfString;
+use setasign\Fpdi\PdfParser\Type\PdfType;
+use setasign\Fpdi\PdfParser\Type\PdfTypeException;
+
+/**
+ * @mixin Mpdf
+ */
+trait FpdiTrait
+{
+	use \setasign\Fpdi\FpdiTrait {
+		writePdfType as fpdiWritePdfType;
+		useImportedPage as fpdiUseImportedPage;
+	}
+
+	protected $k = Mpdf::SCALE;
+
+	/**
+	 * The currently used object number.
+	 *
+	 * @var int
+	 */
+	public $currentObjectNumber;
+
+	/**
+	 * A counter for template ids.
+	 *
+	 * @var int
+	 */
+	protected $templateId = 0;
+
+	protected function setPageFormat($format, $orientation)
+	{
+		// in mPDF this needs to be "P" (why ever)
+		$orientation = 'P';
```

Comment:
This is a very strange behavior of mPDF and I didn't found any other reasonable way to "fix" this.